mk/dependencies/kubebuilder.sh

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
 
+set -e
+
+if [ "$ARCH" == "arm64" ]; then
+  if [ "$OS" == "darwin" ]; then
+    # there is no arm64 build for darwin so use amd64 via rosetta
+    ARCH="amd64"
```

Comment:
this never worked on m1 - https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v2.3.2